eng/build.ps1

```diff
@@ -20,46 +20,78 @@ Param(
 
 function Get-Help() {
   Write-Host "Common settings:"
-  Write-Host "  -subset                   Build a subset, print available subsets with -subset help (short: -s)"
-  Write-Host "  -vs                       Open the solution with VS using the locally acquired SDK. Path or solution name (ie -vs Microsoft.CSharp)"
-  Write-Host "  -os                       Build operating system: Windows_NT, Linux, OSX, or Browser"
-  Write-Host "  -arch                     Build platform: x86, x64, arm, arm64, or wasm (short: -a). Pass a comma-separated list to build for multiple architectures."
-  Write-Host "  -configuration            Build configuration: Debug, Release or [CoreCLR]Checked (short: -c). Pass a comma-separated list to build for multiple configurations"
-  Write-Host "  -runtimeConfiguration     Runtime build configuration: Debug, Release or [CoreCLR]Checked (short: -rc)"
-  Write-Host "  -librariesConfiguration   Libraries build configuration: Debug or Release (short: -lc)"
-  Write-Host "  -verbosity                MSBuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic] (short: -v)"
-  Write-Host "  -binaryLog                Output binary log (short: -bl)"
-  Write-Host "  -help                     Print help and exit (short: -h)"
+  Write-Host "  -arch (-a)                     Build platform: x86, x64, arm, arm64, or wasm."
+  Write-Host "                                 Pass a comma-separated list to build for multiple architectures."
+  Write-Host "                                 [Default: Your machine's architecture.]"
+  Write-Host "  -binaryLog (-bl)               Output binary log."
+  Write-Host "  -configuration (-c)            Build configuration: Debug, Release or [CoreCLR]Checked."
```

Comment:
Doesn't casing matter (at least on Unix)? @safern should it be consistently lower case or upper? We should note that in the help, and fix the help.

Reply:
The casing doesn't matter as we are upper-casing passed-in configurations in the scripts.

Reply:
Right. In a previous PR I made ([35642](https://github.com/dotnet/runtime/pull/35642)) it no longer matters if you pass `--os LINUX` or `--os linux`. It gets handled internally and converted to the casing MSBuild expects.

Reply:
consistently using lower case in examples seems better but I can't put my finger on why so it's probably just me :)